languages/tree-sitter-stack-graphs-python/src/stack-graphs.tsg

```diff
@@ -13,6 +13,7 @@
 ;; ^^^^^^^^^^^^^^^^
 
 global FILE_PATH
+global ROOT_PATH
```

Comment:
If you set a default value here, it makes life a bit easier because you don't have to set `ROOT_PATH` unless you need to.

```suggestion
global ROOT_PATH = ""
```